api_volontaria/apps/donation/serializers.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import serializers, generics
+
+from api_volontaria.apps.donation.models import Donation
+
+
+class DonationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Donation
+        fields = '__all__'
+
+
+class DonationApiCreate(generics.ListCreateAPIView):
+    queryset = Donation.objects.all()
+    serializer_class = DonationSerializer
```

Comment:
All the views should be in the `views` files, only serializers should be in this one.